package io.quizApp.Exceptions;

public class NotFoundException extends RuntimeException {
	public NotFoundException(String message, String cause) {

	}
}
